devel/sift/Portfile

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem 		1.0
+
+# PortGroup		github 1.0
+# github.setup	svent sift 0.9.0 v
+
+name            sift
+version         0.9.0
+platforms       darwin
+categories      sysutils
+license         GPL-3+
+installs_libs   no
+maintainers     esafak
+description     A fast and powerful open source alternative to grep
+long_description sift is an alternative that aims for both speed and flexibility, \
+				adding features while trying to reach (or even surpass) the performance \
+				of the original grep. The additional features include gitignore support, \
+				conditions (e.g., match A only when preceded by B within X lines), \
+				full multi-core support and multiline matching.
+homepage        http://sift-tool.org
+platforms       darwin
+use_zip          yes
+
+master_sites	https://github.com/svent/sift/archive:sift \
+				https://github.com/jessevdk/go-flags/archive:flags \
+				https://github.com/svent/go-nbreader/archive:nbreader \
+				https://raw.githubusercontent.com/golang/crypto/master/ssh/terminal:crypto
+distfiles		v${version}.zip:sift \
+				terminal.go:crypto \
+				util_bsd.go:crypto \
+				util.go:crypto \
+				v1.1.0.zip:flags \
+				master.zip:nbreader
```

Comment:
Is it really necessary to download dependencies this way? I have no knowledge of Go, but shouldn't these libraries be in separate ports and just be linked?